### DISCUSSION TITLE
Add details about Runner OS version in Actions environment

### DISCUSSION THREAD

**Discussion by sagarchikane** [actions/virtual-environments#7879] [Ideas]
Can we have below highlighted values readily available as part of [Default Environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) for a Github Action
<img width="885" alt="RunnerDetails" src="https://github.com/actions/runner-images/assets/75995353/cc4895d5-16c7-461c-aa0e-ef4897459c22">

[This thread](https://github.com/actions/runner/discussions/1399) mentions something about fetching the version of the self hosted runner but it isn't clear about which script to execute

My use-case involves fetching the version for Runner image.

One way to do that would be to listen for that job to complete and then use Github CLI/API commands to fetch the action logs and then grep the logs to find the said version. This seems like quite the long process to fetch something that could be readily available as an environment variable for developers to directly use.

**Comment by vpolikarpov-akvelon** [comment]
Hey @sagarchikane. Image OS and version are available as environment variables `$ImageOS` and `$ImageVersion`. Is this what you are looking for?

**Reply by sagarchikane** [reply]
Thanks for the reply @vpolikarpov-akvelon , yes this is what I was looking for. Couldn't find these in docs directly which caused the confusion

**Reply by janbrasna** [reply]
Issue raised for documenting there: https://github.com/github/docs/issues/40108

(Thanks for this nugget of information anyways, so useful!! 🚀)

**Reply by masbin387-sys** [reply]
#!/bin/sh

sed -i "s/\$SENTINEL_QUORUM/$SENTINEL_QUORUM/g" /etc/redis/sentinel.conf
sed -i "s/\$SENTINEL_DOWN_AFTER/$SENTINEL_DOWN_AFTER/g" /etc/redis/sentinel.conf
sed -i "s/\$SENTINEL_FAILOVER/$SENTINEL_FAILOVER/g" /etc/redis/sentinel.conf

exec docker-entrypoint.sh redis-server /etc/redis/sentinel.conf --sentinel#!/bin/sh

sed -i "s/\$SENTINEL_QUORUM/$SENTINEL_QUORUM/g" /etc/redis/sentinel.conf
sed -i "s/\$SENTINEL_DOWN_AFTER/$SENTINEL_DOWN_AFTER/g" /etc/redis/sentinel.conf
sed -i "s/\$SENTINEL_FAILOVER/$SENTINEL_FAILOVER/g" /etc/redis/sentinel.conf

exec docker-entrypoint.sh redis-server /etc/redis/sentinel.conf --sentinel
